BUILDING_MULTI_ARCH.md

```diff
@@ -0,0 +1,31 @@
+# BUILDING MULTI-ARCHITECTURE IMAGES
+
+The Docker images for OpenKruise can be built by running:
+
+```
+make docker-build
+```
+
+However, there are several options and requirements for building these images, which are explained here.
+
+## BUILDERS
+
+We use `docker buildx` to build images for multiple architectures. At the moment, this is considered an experimental feature by Docker, and as such it requires access to experimental features to be enabled in the daemon.json file.
+
+By default, this will use BuildKit and QEMU emulation support to build the image for each architecture locally. However, for faster building, it is possible to use native nodes to build each architecture, either simple remote Docker instances or a Kubernetes cluster. For information on how to set this up, please see here:
+
+<https://docs.docker.com/engine/reference/commandline/buildx_create/>
```

Comment:
This blog may be better https://www.docker.com/blog/multi-arch-images

Reply:
Added that, leaving the command reference in as another link.